### PR TITLE
2018-01-26-Security-Automation-Lessons-Learned.md

### PR DESCRIPTION
**Pull request by cneill**
__Bio__

Charles Neill is a Security Engineer at Rackspace, where he tests Rackspace products and develops tools to make security testing easier. He's worked on a number of open source security projects such as [DefectDojo](https://github.com/DefectDojo/django-DefectDojo) (an issue tracking tool focused on security defects) and [Syntribos](https://github.com/openstack/syntribos) (an API security testing framework built for OpenStack).

__Publish Date:__ 1-26-18

__Social Media__

[Twitter](https://twitter.com/ccneill)
[Github](https://github.com/cneill)

In a comment on the pull request, include the following information and check off the list.

 - [x] Mini bio
  - Soft capped to 75 words
 - [x] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [x] Social media contact info
 - [x] Redaction: yes
  - Allow redaction, editing or structure changes by technical writer staff
 - [x] Comments: yes
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-ed3eb23238/)

File: _posts/2018-01-26-Security-Automation-Lessons-Learned.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+affected endpoints (common for host scanning tools)? Is there a file or git
+commit that is associated with multiple findings? Thinking about this early
+can save you headaches in the long term.
+
+### 4) Clarify your outputs
+
+What are you looking to get out of this "security automation" of yours? Faster
+test results? More fine-grained analysis of defects? Cross-referencing issues
+across different projects? PDF reports for leadership? Come up with a list of
+outputs you _could_ produce, then ask yourself, "why?" and/or "so what?". Let's
```
Comment (catlook) [inline _posts/2018-01-26-Security-Automation-Lessons-Learned.md]: ... produce, and then ...
... "why?" or "so what?" ...

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-9ef8705210/)

**Comment by cneill** [comment]
Great, thanks for all the comments, Cat! I clearly still haven't figured out how to use a comma 😛 Someday!

**Pull request merged**